Saturation/Splitter.cpp

```diff
@@ -1082,6 +1089,20 @@ bool Splitter::doSplitting(Clause* cl)
 {
   CALL("Splitter::doSplitting");
 
+  static bool hasStopped = false;
```

Comment:
Could we make this a class variable? We're trying to remove static's in another branch.

Reply:
Actually, we'll just make it thread-local, ignore that.

Reply:
What is going to be the price of thread-local variables? And in a version which does not aim to be multi-threaded?